sample/EasyTables/CSharp/QueueToInItemToOutItem/run.csx

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json.Linq;
+
+public static void Run(string input, Item queriedItem, out Item newItem)
+{
+    newItem = new Item();    
```

Comment:
You're modifying the queried item, and adding a new item? A lot going on in here. Probably you mean for this to be down in the test folder, since this seems "testy" rather than user sampley.